src/sentry/static/sentry/app/views/eventsV2/table/index.tsx

```diff
@@ -75,11 +75,23 @@ class Table extends React.PureComponent<TableProps, TableState> {
   };
 
   fetchData = () => {
-    const {eventView, organization, location, setError} = this.props;
+    const {organization, location, setError} = this.props;
+    let {eventView} = this.props;
 
     if (!eventView.isValid()) {
       return;
     }
+
+    const hasAggregates = eventView.getAggregateFields().length > 0;
+    const hasEventIdColumn = eventView.getFields().includes('id');
+
+    if (!hasAggregates && !hasEventIdColumn) {
+      eventView = eventView.withNewColumn({
+        kind: 'field',
+        field: 'id',
+      });
+    }
```

Comment:
The API endpoint will automatically append the event id when a result set has no aggregates, as we needed it in the past to create links. You shouldn't need this.

Reply:
Interesting. I'll revert this hunk.

Reply:
Reverted in https://github.com/getsentry/sentry/pull/19332/commits/49d520e3beef265cb5a4438f619f57fd4955cbc0